src/activities/ScoringEntity.js

```diff
@@ -27,6 +27,10 @@ export class ScoringEntity extends Entity {
 		return this._entity && this._entity.hasActionByName(Actions.activities.scoreOutOf.update);
 	}
 
+	equals(scoring) {
+		return String(this.scoreOutOf()) === scoring.scoreOutOf;
```

Comment:
Is it ever possible that `scoring.scoreOutOf` might not be a string?

Reply:
Nope, it's initialized like this: `entity.scoreOutOf() ? entity.scoreOutOf().toString() : ''` and otherwise comes from an input field, so will always be a string.

Reply:
Actually, I think I'm wrong, as the input now is a `d2l-input-number` which is a number type, not string. I'll put up another PR, thanks for asking that!